service/impl/MessageServiceImpl.go

```diff
@@ -0,0 +1,72 @@
+package impl
+
+import (
+	"TinyTikTok/dao"
+	"TinyTikTok/models"
+	"log"
+	"strconv"
+	"time"
+)
+
+type MessageServiceImpl struct {
+}
+
+func (messageServiceImpl MessageServiceImpl) SendMessage(toUserid string, fromUserId int64, content string) error {
+
+	//将toUserid从String转换成Int64
+	toUserId, err := strconv.ParseInt(toUserid, 10, 64)
+	if err != nil {
+		log.Printf("toUserid从String转换成Int64失败 %v", err)
+		return err
+	}
+
+	//准备储存到数据库中的数据
+	var message = models.Message{
+		CommonEntity: models.CommonEntity{
```

Comment:
这里我理解的是可以直接这样写
```suggestion
		CommonEntity: models.NewCommonEntity(),
```
因为对于每一条数据来说，其自身 `id` 应该是影响不大的，但是不能重复。

Reply:
嗯嗯，因为我的数据库是自增的，就没用雪花算法，下个数据库尝试用一下